Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/BoundsControl/Visuals/ProximityEffect/ProximityEffect.cs

```diff
@@ -12,8 +12,8 @@ namespace Microsoft.MixedReality.Toolkit.UI.Experimental
     /// Scaling is done on three different stages: far / medium and close proximity whereas material switching 
     /// will only be done on close proximity.
     /// </summary>
-    [Serializable]
-    public class ProximityEffect 
+    [CreateAssetMenu(fileName = "ProximityEffect", menuName = "Mixed Reality Toolkit/Bounds Control/ProximityEffect")]
```

Comment:
weird that the others are "Scale Handles" and this is "ProximityEffect" without space